src/Resources/Locales/en.json

```diff
@@ -1392,11 +1395,20 @@
                 "messages": {
                     "noColor": "Not set",
                     "loading": "Loading historized data...",
-                    "noData": "No historized data found in selected time frame.",
+                    "noData": "No historized data found in the selected time range.",
                     "noSeries": "No twin selected to query, please add twin properties to view historized data."
                 }
             },
             "noSeriesDescription": "To view data history for this twin, select a property to view."
         }
+    },
+    "dataHistoryErrorHandlingWrapper": {
+        "errors": {
+            "genericNetwork": "Cluster cannot be connected which may be due to invalid url, stopped service or a network error.",
+            "genericBadRequest": "Bad request exception occured.",
+            "generalBadRequestMessage": "Request is invalid and cannot be executed.",
```

Comment:
Is this because of something the user did?

Reply:
To capture General_BadRequest error code from Kusto: https://learn.microsoft.com/en-us/azure/data-explorer/kusto/api/netfx/kusto-data-client-errors#request-exceptions

The content of the messages what the error message says in the response.
![image](https://user-images.githubusercontent.com/45217314/216200541-7f406290-b145-4f51-9f3c-bb05585c939b.png)